icu4c/source/test/intltest/intltest.cpp

```diff
@@ -2428,6 +2429,27 @@ UChar *IntlTest::ReadAndConvertFile(const char *fileName, int &ulen, const char
     return retPtr;
 }
 
+#if !UCONFIG_NO_BREAK_ITERATION
+UBool LSTMDataIsBuilt() {
```

Comment:
Suggestion: Add these methods to a special subclass of `IntlTest` in `itrbbi.h`, similar to `IntlTestWithFieldPosition` in `itformat.h`.

Reply:
The problem is we need to access these methods on any classes which internally use rbbi, not just the test on rbbi itself. For example, we need to call it in TransliteratorTest::TestThai inside transtst.cpp  .